etl/churn.kp/knowledge.md

```diff
@@ -326,6 +329,10 @@ def convert(d2v, week_start, datum):
     out["usage_hours"] = (datum.usage_seconds / SECONDS_IN_HOUR) if datum.usage_seconds is not None else 0.0
     out["squared_usage_hours"] = out["usage_hours"] ** 2
     
+    # d.get does not default to 0, so make sure that its an int here
```

Comment:
what is `d.get` referring to in this comment? It appears that nulls are handled in the `current_week.na.fill(0, ["total_uri_count", "unique_domains_count"])` line below.